src/dash-adapter.js

```diff
@@ -223,6 +223,9 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
   _isStartOver: boolean = true;
```

Comment:
add a configuration option to enable/disable the manifest static switch
see an example here about how to add a config param
https://github.com/kaltura/playkit-js-dash/commit/99894f37f7778e5c18f0e08c85c933543462b15c

Reply:
we want to add a flag to the player configuration so only customers who set true for that flag will use this logic

so that means we need to:
1. add a new config variable in default-config.json, with default value false. the adapter config will now be initialized with this value by default
2. add code in createAdapter to override the default config if the user added this value to his config
3. put an if condition around the logic in registerResponseFilter that uses the new config value (similar to how we use trackEmsgEvents)
4. check that when the value is false we don't invoke the new logic
5. open a PR in playkit-js to add the new config parameter to the documentation, like in this PR https://github.com/kaltura/playkit-js/pull/668/files